Source/AviationWeather.Test/Source/framework.h

```diff
@@ -46,6 +46,11 @@ template<> inline std::wstring ToString<uint16_t>(const uint16_t& t) { RETURN_WI
 template<> inline std::wstring ToString<uint16_t>(const uint16_t* t) { RETURN_WIDE_STRING(t); }
 template<> inline std::wstring ToString<uint16_t>(uint16_t* t)       { RETURN_WIDE_STRING(t); }
 
+// aw::metar::visibility
+template<> inline std::wstring ToString<aw::metar::visibility>(const aw::metar::visibility& t) { /* ugh.... */ RETURN_WIDE_STRING(1); }
```

Comment:
What are these `ToString` templates for, and why does line 50 have `RETURN_WIDE_STRING(1)`?